protostar/commands/init/project_creator/adapted_project_creator.py

```diff
@@ -1,32 +1,11 @@
 from pathlib import Path
 
 from protostar.commands.init.project_creator._project_creator import ProjectCreator
-from protostar.protostar_toml.io.protostar_toml_writer import ProtostarTOMLWriter
-from protostar.self.protostar_directory import VersionManager
-from protostar.io.input_requester import InputRequester
 from protostar.git import Git
 
 
 class AdaptedProjectCreator(ProjectCreator):
```

Comment:
This `AdaptedProjectCreator` is almost the same as  `ProjectCreator` right now.  I'd opt out from inheritance and do some method composition, or extract the common parts into methods.

Reply:
Yep, inheritance shouldn't be used to share low level code. However, I'd say this is out of the scope of this PR.

Reply:
Please create an issue or leave a TODO before merging

Reply:
https://github.com/software-mansion/protostar/issues/1091